.github/workflows/build.yml

```
@@ -27,6 +27,10 @@ jobs:
         run: |
           ./scripts/lint-copy-mod-doc.sh
 
+      - name: check for executable .lean files
+        run: |
+          ! find . -name *.lean -type f -executable | grep -q .
```

```suggestion
          ! find . -name '*.lean' -type f -executable | grep .
```

1. Stars need to be escaped, otherwise bash interprets it as a wildcard.
2. It is useful to see which files where executable and caused the linter to fail, so please drop the `-q`.

---

Thanks!